Site1.Master.cs

```diff
@@ -11,7 +11,45 @@ public partial class Site1 : System.Web.UI.MasterPage
     {
         protected void Page_Load(object sender, EventArgs e)
         {
+            try
+            {
+                if (Session["role"].Equals(""))
+                {
+                    LinkButton1.Visible = true; //user login link button
```

Comment:
If you have proper naming conventions, you would not need to have to comment every single line of code to tell people what it means. 
```suggestion
                    lbUserLogin.Visible = true;
```

Reply:
The same goes for the rest of the code.